packages/webapp/src/components/Card/NotificationCard/NotificationCard.jsx

```diff
@@ -37,6 +38,7 @@ export function PureNotificationCard({
   let Icon;
   // The "context" can indicate that a particular type of task icon is appropriate.
   if (context?.task_translation_key) Icon = getTaskTypeIcon(context.task_translation_key);
+  if (context?.icon_translation_key) Icon = getNotificationTypeIcon(context.icon_translation_key);
```

Comment:
I was thinking about doing that but was afraid that the refactoring would break the code as `task_translation_key` was used all throughout the app and the database it seems. It will be safer to make a separate ticket that only focuses on this refactoring (involves migration files)

Reply:
Right. Good point.